.github/CONTRIBUTING.md

```diff
@@ -13,13 +13,13 @@ project, you agree to abide by its terms.
 
 ## General workflow
 
-This is the workflow for contributing changes to Emogrifier:
+This is the workflow for contributing changes to this project::
 
-1. [Fork the Emogrifier Git repository](https://guides.github.com/activities/forking/).
-2. Clone your forked repository and
+1. [Fork the git repository](https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project).
+2. Clone your forked repository locally and
    [install the development dependencies](#install-the-development-dependencies).
 3. Add a local remote "upstream" so you will be able to
-   [synchronize your fork with the original Emogrifier repository](https://help.github.com/articles/syncing-a-fork/).
+   [synchronize your fork with the original repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).
```

Comment:
I know this doesn't relate specifically to this PR, but...

I think both 'remote' and 'upstream' should be in inverted-ticks, and I'm not sure 'local' belongs.  'A local remote 
"upstream"' does not appear to make any sense.  Though I don't fully understand what's going on here, and not sure I ever did that step.

The link given provides a link in turn for adding a `remote` `upstream", so an extra link for that probably isn't needed (though might be considered).

Reply:
I think I'll just remove that step. GitHub has added some UI for that anyway in the meantime.